### PR TITLE
Upload CSV

### PR DESCRIPTION
**Pull request by jonahheeren**

### CODE REVIEW

**Comment by jonahheeren** [comment]
Not completely perfect but I wanted to get this up so @vritant24 would have my UI changes. Most of the issues come from the UI transitions not being correct after clicking 'Select another location'. Passing the data to the algorithm and graph should all be solid.

File: TDPAlg.js [hunk 1]
```diff
@@ -134,12 +135,17 @@ module.exports.calc = function(_pondVolSmallest, _pondVolLargest, _pondVolIncrem
...
-
       resolve(allAnnuals);
-
+    })
     });
+};
 
+function pullData(_locationId, stream) {
+  return new Promise(function(resolve, reject) {
+    if(_locationId && _locationId !== "undefined")
```
Comment (drewatk) [inline TDPAlg.js]: Maybe this should be  `if(Number.isInteger(_locationId))`

File: public/app.js [hunk 2]
```diff
@@ -25,3 +25,14 @@ $('#form-submit').click(function() {
     toggleText = 1;
   });
 });
+
+$('#upload_submit').click(function() {
+  $('#divider').fadeOut('fast', function() {
```
Comment (drewatk) [inline public/app.js]: how come this is nested unlike on line 10?

File: public/graph.js [hunk 3]
```diff
@@ -8,21 +8,22 @@ graphData = [];
...
   }
 
   $.ajax({
     type: 'POST',
     url: '/calculate',
-    data: JSON.stringify(data),
-    contentType: "application/json",
-    dataType: 'json',
+    data: formData,
+    contentType: false,
```
Comment (drewatk) [inline public/graph.js]: I think this should be "multipart/form-data"

File: public/app.js (hunk 2 above)
Comment (jonahheeren) [inline public/app.js]: The stuff at 10 I only indented, I didn't write it. Mine is all nested so it creates a flowing animation and doesn't choppily move things around executing those statements asynchronously.

File: public/graph.js (hunk 3 above)
Comment (jonahheeren) [inline public/graph.js]: Technically you are right and it should be multipart/form-data, but when I switch it, the request fails saying there is no multipart boundary. So I'm not sure why setting the content type to false works but it was recommended on stack overflow as well. ¯\_(ツ)_/¯

**Pull request merged**